app/controllers/search_controller.rb

```diff
@@ -18,11 +18,12 @@ def search
                                   "%#{params[:q].downcase}%",
                                   "%#{params[:q].downcase}%",
                                   "%#{params[:q].downcase}%")
-    repositories_by_categories = []
-    Category.where('lower(name) LIKE ?', params[:q].downcase).each do |cat|
-      repositories_by_categories << cat.repository
-	  end
-    @repositories = repositories_by_attributes + repositories_by_categories
+    if category = SLUG_TO_OLD_CATEGORY[params[:slug]]
```

Comment:
I don't think this is correct for search since there isn't a param called `:slug`. I'm not sure how to fix the weird category stuff for search honestly